api/chaos/experimentation/v1/experimentation.proto

```diff
@@ -54,9 +54,9 @@ service ExperimentsAPI {
     };
     option (clutch.api.v1.action).type = READ;
   }
-  rpc DeleteExperiments(DeleteExperimentsRequest) returns (DeleteExperimentsResponse) {
+  rpc StopExperiments(StopExperimentsRequest) returns (StopExperimentsResponse) {
     option (google.api.http) = {
-      post : "/v1/experiments/delete"
+      post : "/v1/experiments/stop"
```

Comment:
@Augustyniak your pending change will need to be adjusted to point to this URI

Reply:
makes sense 👍